cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractDistributedProgramRunner.java

```diff
@@ -291,9 +293,9 @@ public TwillController launch(TwillApplication twillApplication, Iterable<String
                 Collections.singletonList(HBaseTableUtilFactory.getHBaseTableUtilClass()),
                 getKMSSecureStore(cConf), extraDependencies);
 
-              Iterable<String> yarnAppClassPath = Arrays.asList(
-                hConf.getStrings(YarnConfiguration.YARN_APPLICATION_CLASSPATH,
-                                 YarnConfiguration.DEFAULT_YARN_APPLICATION_CLASSPATH));
+              Iterable<String> yarnAppClassPath = Splitter.on(",").trimResults().split(
```

Comment:
I think ``hConf`` has ``getTrimmedString*`` which might make this a little succint